manifests/hourly.pp

```diff
@@ -6,35 +6,33 @@
 # Examples
 #
 #   # Set up hourly logrotate jobs
-#   include logrotate::hourly
+#   class { 'logrotate':
+#     manage_cron_hourly => true,
+#   }
 #
 #   # Remove hourly logrotate job support
-#   class { 'logrotate::hourly':
-#     ensure => absent,
+#   class { 'logrotate':
+#     manage_cron_hourly => true,
+#     ensure_cron_hourly => absent,
 #   }
 class logrotate::hourly (
-  Enum['present','absent'] $ensure = 'present'
```

Comment:
Removing this parameter is a breaking change, we should add matching labels for this PR.

Reply:
I've added the label.

Reply:
Probably should remove parameter's `()` too, since no attributes anymore?